buildSrc/src/main/kotlin/ReleaseScripts.kt

```diff
@@ -411,12 +432,16 @@ private fun validateVersion(
     return schemaExists && sameMajor
 }
 
-private fun commitChangesInFile(gitDirectory: File, absoluteFilePath: File, commitMessage: String) {
+private fun commitChangesInFiles(gitDirectory: File, commitMessage: String, vararg absoluteFilePaths: File) {
     val git = createGitInstance(gitDirectory)
-    val relativePath = gitDirectory.toPath().relativize(absoluteFilePath.toPath()).toString()
 
-    git.commit()
-        .setOnly(relativePath)
+    val commit = absoluteFilePaths
+        .fold(git.commit()) { commit, file ->
+            val relativePath = gitDirectory.toPath().relativize(file.toPath()).toString()
+            commit.setOnly(relativePath)
+        }
```

Comment:
Do we to relativize these? `git commit --only` supports absolute paths too (I checked through terminal).

Reply:
For whatever reason it doesn't work in the JGit library
![image](https://github.com/VirtuslabRnD/pulumi-kotlin/assets/34098778/415707b5-06fa-4885-b030-8d6e1292dcf6)